src/components/AirdropBalanceCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import { AnimatedCircularProgress } from 'react-native-circular-progress';
+
+import { AirdropCarouselCardData } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+interface Props {
+  data: AirdropCarouselCardData;
+}
+
+export const AirdropBalanceCard: FC<Props> = ({ data }) => (
+  <View>
+    <Text style={styles.header}>{data.header}</Text>
+    <AnimatedCircularProgress
+      size={155}
+      width={8}
+      fill={data.circleFillPercentage}
+      tintColor={palette.textSecondary}
+      backgroundColor={palette.lightGrey}
+      lineCap="round"
+      style={styles.circle}
+    >
+      {() => (
```

Comment:
Why is a function here? What does it do?

Reply:
library expects a function here :)